src/app/feature/theme/theme-service/theme.service.ts

```diff
@@ -28,6 +28,17 @@ export class ThemeService {
 
     this.currentTheme = this.getCurrentTheme();
     this.applyCSSVariablesForTheme(this.currentTheme);
+    this.applyTheme(this.currentTheme.name);
+  }
+
+  public applyTheme(themeName: string) {
+    document.body.classList.add("theme-" + themeName);
```

Comment:
nit(non-blocking) - it might be tidier to just use a data attribute (e.g. data-theme='default') to save adding and removing classes each time